docs/ELRS_RECEIVERS.md

```diff
@@ -51,6 +70,7 @@ Note: In order to send RC channels over the serial connection, change the Rx Snd
 
 ## Flashing ##
 
+- Download the firmware for your receiver [here](https://github.com/olliw42/mLRS/tree/main/firmware/pre-release-esp).
```

Comment:
there is the slight inconvenience that these are the latest pre-release versions (even number), some users may prefer the offical version ... not sure how to handle that 

note: it currently are odd numbers ... I think that's my mistake, should be even ...

Reply:
Could link to the release page but currently 1.2 has no ESP stuff in the source code archive.

Reply:
maybe just adding some text to at least clarify what one gets ...

Reply:
Not 100% sure what you mean - would something like 'Download the corresponding .bin for your receiver [here]' be better?

Reply:
I'm not sure I'm 100% sure myself what I mean :)
just straight from one nerve cell: Download the firmware for your receiver. The latest pre-release you find here. For the latest offical release version look here.

Reply:
Okay, added that it.

Reply:
NICE! a straightforward list is a good idea